feconf.py

```diff
@@ -47,6 +47,7 @@
 else:
     raise Exception('Invalid platform: expected one of [\'gae\']')
 
+CLASSIFIERS_PKG = 'core.domain.classifier'
```

Comment:
Will remove it and would use CLASSIFIERS_DIR instead once the alternative approach works.